src/common/tokenHelpers.ts

```diff
@@ -35,8 +40,10 @@ export function makeMetaTokens(tokens: EdgeTokenMap): EdgeMetaToken[] {
 
 export const getTokenIdFromCurrencyCode = (
   currencyCode: string,
+  mainnetCurrencyCode: string,
   allTokensMap: EdgeTokenMap
-): string | undefined => {
+): EdgeTokenId | undefined => {
+  if (currencyCode === mainnetCurrencyCode) return null
```

Comment:
The ripple plugin uses `!= null` to deal with the return value, which is a big problem now that we have a separation between `null` and `undefined`. Each of the 4 places needs to be updated to `!== undefined` or whatever other logic would be needed.

Reply:
I only found one spot to change. A few usages of != null should only allow strings through.